tests/testAutopas/TimerTest.cpp

```diff
@@ -18,5 +18,5 @@ TEST(TimerTest, testTimer) {
   usleep(10e3);
   time = timer.stop();
   // time should be close to 10ms
-  ASSERT_NEAR(time, 10e-3, 1e-3);
+  ASSERT_NEAR(time, 10e-3, 5e-3);
```

Comment:
The asserts appear to be a bit weak but I do not know how "accurate" such timers are in general. According to my machine, up to 1e-4 is ok here and 1e-7 for the `ASSERT_LE` above.

Reply:
I just noticed that this test sometime fails. so I changed it slightly, to at least check the right magnitude.